static/modal_style.css

```diff
@@ -0,0 +1,224 @@
+/* Modals */
+
+.modal-1 {
+  width: 100%;
+  height: 850px;
+  background-color: #3c3a39;
+  color: #fff5e1;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Roboto', sans-serif;
+  border: 0;
+}
+
+.modal-1 div {
+  display: flex;
+  justify-content: right;
+  padding: 24px;
+  color: #6f6c6b;
+  font-weight: 400;
+}
+
+.modal-1 ul {
+  list-style-type: none;
+  padding: 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+}
+
+.modal-1 ul li {
+  padding: 8px 16px;
+  border-bottom: #6f6c6b solid 1px;
+}
+
+.modal-1 ul li a {
+  color: #fff5e1;
+  text-decoration: none;
+}
+
+.modal-2 {
+  width: calc(100% - 96px);
+  padding: 24px;
+  margin: 24px 24px 0 24px;
+  background-color: #fff;
+  color: #344563;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Roboto', sans-serif;
+  border: 0;
+}
+
+dialog::backdrop {
+  background-color: rgb(163, 159, 159);
+  opacity: 0.9;
+}
+
+.all .text_side {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 24px 24px 24px 0;
+}
+
+.text_side .buttons {
+  height: 100%;
+}
+
+.modal-2 div {
+  color: #6f6c6b;
+  font-weight: 400;
+}
+
+.modal-2 div.wrp {
+  display: flex;
+  justify-content: right;
+  padding: 0;
+  margin: 0;
+}
+
+.modal-2 .close-1 {
+  padding-top: 24px;
+}
+
+.modal-2 .all {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.all h2 {
+  font-family: 'Roboto', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #172b4d;
+}
+
+.all div.image {
+  display: flex;
+  justify-content: left;
+  padding: 0;
+  margin: 24px 0 24px 0;
+}
+
+.all div.image img {
+  width: 100%;
+  max-height: 400px;
+  min-width: 270px;
+}
+
+.modal-2 .links {
+  padding: 24px 0;
+}
+
+.all ul li {
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #3c3a39;
+}
```

Comment:
- [ ] The font weight to the list of languages has not been implemented in your design, kindly do so.

Given Template
![image](https://github.com/wacoo/Portfolio-Project/assets/81932478/4bc30fa4-d4b4-4dca-ac2f-8472208f7414)

Your design
![image](https://github.com/wacoo/Portfolio-Project/assets/81932478/baf02340-73c7-4f9c-9518-e3f3cd491f3f)

#### Kindly make your text bolder, and use given font- family which is `poppins` and not inter` which is what has been applied here. This applies for the desktop version as well :red_circle: